src/app/embed/embed.component.ts

```diff
@@ -32,15 +35,18 @@ export class EmbedComponent implements OnInit {
   subscribeTarget: string;
   artworkUrl: string;
   feedArtworkUrl: string;
+  pymId?: string;
 
   constructor(private route: ActivatedRoute, private adapter: MergeAdapter) {}
 
   ngOnInit() {
     this.route.queryParams.forEach(params => {
+      this.pymId = params[PYM_CHILD_ID_PARAM];
       this.adapter.getProperties(params).subscribe(props => {
         this.assignEpisodePropertiesToPlayer(props);
       });
     });
+    this.setEmbedHeight();
```

Comment:
Should this be inside the queryParams forEach, to make sure the `pymId` is set before making the initial call?

Though in reality, the `queryParams.forEach` observable may alway run sync.